dev/tasks/tasks.yml

```diff
@@ -240,220 +242,208 @@ tasks:
   #   the same feedstock as the dependency matrix is the same for them as
   #   Python and the OS are the main dimension. The R package `r-arrow` is
   #   an independent feedstock as it doesn't have the Python but the
-  #   R dimension. To limit the number of CI jobs, we are building `r-arrow`
-  #   with the Python 3.7 jobs.
+  #   R dimension.
   # * The files in `dev/tasks/conda-recipes/.ci_support/` are automatically
   #   generated and to be synced regularly from the feedstock. We have no way
   #   yet to generate them inside the arrow repository automatically.
 
-  conda-linux-gcc-py37-cpu-r40:
+  conda-linux-x64-cpu-r41:
```

Comment:
The overall diff in this file is pretty busted. Happy to explain if questions arise (also see individual commits)